src/python/grpcio_tests/tests_aio/unit/server_test.py

```diff
@@ -230,6 +235,18 @@ async def _invalid_trailing_metadata(self, request, context):
             trailing_metadata=({"error": ("error1", "error2")}),
         )
 
+    async def _abort_and_slow_finalization(self, request, context):
+        try:
+            try:
+                await context.abort(grpc.StatusCode.INVALID_ARGUMENT)
+            finally:
+                # server.stop() should either wait for this to complete (graceful),
+                # or cancel this and finish execution
+                await asyncio.sleep(test_constants.SHORT_TIMEOUT)
+        finally:
+            self.abort_and_slow_finalization_finished = True
```

Comment:
**n.b.** Prior to this patch, this line was never executed because `server.stop()` completed much earlier, never waiting for the in-progress async tasks to complete.